clientUtils/formatValue.ts

```diff
@@ -1,92 +1,199 @@
-import { d3Format } from "./Util.js"
+import { FormatSpecifier } from "d3-format"
+import { createFormatter } from "./Util.js"
 
 export interface TickFormattingOptions {
     numDecimalPlaces?: number
     unit?: string
-    noTrailingZeroes?: boolean
-    noSpaceUnit?: boolean
-    numberPrefixes?: boolean
-    shortNumberPrefixes?: boolean
+    trailingZeroes?: boolean
+    spaceBeforeUnit?: boolean
     showPlus?: boolean
+    numberAbreviation?: "short" | "long" | false
```

Comment:
I think it should be `Abbreviation` (double b)? Although it's a good way to ab~b~reviate it, so I hope it [catches on](https://en.wikipedia.org/wiki/HTTP_referer#Etymology).